src/r7insight.js

```diff
@@ -392,29 +396,30 @@
         createLogStream: _createLogStream,
         to: _getLogger,
         destroy: _destroyLogStream,
-        log: function() {
+        log: function () {
             for (var k in loggers)
                 loggers[k].log.apply(this, arguments);
         },
-        warn: function() {
+        warn: function () {
             for (var k in loggers)
                 loggers[k].warn.apply(this, arguments);
         },
-        error: function() {
+        error: function () {
             for (var k in loggers)
                 loggers[k].error.apply(this, arguments);
         },
-        info: function() {
+        info: function () {
             for (var k in loggers)
                 loggers[k].info.apply(this, arguments);
         }
     };
 
     function validate_region(region) {
```

Comment:
probably better name for this function now is something like `get_region_prefix`

Reply:
agreed. The naming implies no changes (which used to be the case) now it mutates the parameter. Mutation should be removed in either case.

Reply:
Its not normal naming in js to use underscores; but 🤷  your unloved code base.